admin/shared/form/interceptor/form-errors.interceptor.ts

```diff
@@ -35,27 +38,41 @@ export class FormErrorsInterceptor implements NestInterceptor {
 
     return next.handle().pipe(
       catchError(error => {
-        const parsedError = JSON.parse(error.message) as Array<{
-          property: string;
-          constraints: string[];
-        }>;
-        // In case of validation error, we render the redirect with the flashed errors and DTO
-        return of(parsedError).pipe(
-          map(validationErrors => {
-            return validationErrors.reduce(
-              (acc, currentValidationError) => ({
-                ...acc,
-                [currentValidationError.property]:
-                  currentValidationError.constraints,
+        let errorMessage;
+        try {
+          errorMessage = JSON.parse(error.message);
+        } catch (e) {
+          return next.handle();
+        }
+
+        const formValidationErrors = plainToClass(FormValidationErrorsDto, {
+          message: errorMessage,
+        });
+
+        return from(validate(formValidationErrors)).pipe(
```

Comment:
question: why are you validating the object within a rxjs event queue instead of using classic imperative programming ?